kafka/Dockerfile

```diff
@@ -3,8 +3,18 @@ FROM stackable/image/java-base AS builder
 
 ARG KCAT
 
-RUN microdnf install -y zlib-devel openssl-devel cyrus-sasl-devel libcurl-devel && \
-    microdnf install -y tar which wget zlib gcc-c++ make cmake && \
+RUN microdnf install -y \
+    cmake \
+    cyrus-sasl-devel \
+    gcc-c++ \
+    libcurl-devel \
+    make \
+    openssl-deve \
+    tar \
+    wget \
+    which \
+    zlib \
+    zlib-devel && \
```

Comment:
**praise**: I much prefer the multiline separation :smile:

Reply:
Looking at this...isn't there a typo? openssl-deve -> openssl-devel?

Reply:
Indeed there is. I didn't notice it (but it would have been picked up during the next build).

Reply:
I'll take another look at it. Maybe I committed something unintentionally here as Kafka is the next thing on my list to look at. Maybe this was meant to go on another branch.